code/wde/WaveletDensityEstimator/Compute_wde_2D_Distributed.m

```diff
@@ -82,7 +82,7 @@
 
 % Loop through the shapes, resize them approriately, and estimate the
 % coeffients and densities and store them. 
-for i = 1 : 5
+for i = 1 : 1

```

Comment:
Put a variable for the second argument of this loop so you just change the variable number to change the loop